scripts/monitoring/cron-send-aws-instance-health.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+
+""" Check aws instance health """
+
+#pylint: disable=invalid-name
+#pylint: disable=line-too-long
+#pylint: disable=wrong-import-position
+
+import logging
+logging.basicConfig(
+    format='%(asctime)s - %(relativeCreated)6d - %(levelname)-8s - %(message)s',
+)
+logger = logging.getLogger()
+logger.setLevel(logging.WARN)
+
+import argparse
+import os
+import time
+
+# Our jenkins server does not include these rpms.
+# In the future we might move this to a container where these
+# libs might exist
+#pylint: disable=import-error
+from openshift_tools.monitoring.metric_sender import MetricSender
+from openshift_tools.cloud.aws.base import Base
+
+def parse_args():
+    """ parse the args from the cli """
+    logger.debug("parse_args()")
+
+    parser = argparse.ArgumentParser(description='AWS instance health')
+    parser.add_argument('-v', '--verbose', action='store_true', default=None, help='Verbose?')
+    parser.add_argument('-vv', '--verbose2', action='store_true', default=None)
+    parser.add_argument('--aws-creds-profile', required=False, help='The AWS credentials profile to use.')
+    return parser.parse_args()
+
+def send_metrics(problems):
+    """ send data to MetricSender"""
+    logger.debug("send_metrics(problems)")
+
+    ms_time = time.time()
+    ms = MetricSender()
+    logger.info("Send data to MetricSender")
+
+    ms.add_metric({'openshift.cloud.aws.problems.InstanceStatus': problems['InstanceStatus']})
+    ms.add_metric({'openshift.cloud.aws.problems.SystemStatus': problems['SystemStatus']})
+    ms.add_metric({'openshift.cloud.aws.problems.Events': problems['Events']})
```

Comment:
Consider renaming these to:

`aws.ec2.instance.instance_status`
`aws.ec2.instance.system_status`
`aws.ec2.instance.events`